css/global.less

```diff
@@ -21,6 +21,15 @@ body.custom-background-size {
     background-size:         cover !important;
 }
 
+/* Links */
```

Comment:
Minor: There is already a section for links at the bottom of this file. It makes sense to keep styles together that are for the same thing.

Reply:
Agreed

Reply:
moved